pkglistgen/group.py

```diff
@@ -312,6 +313,37 @@ def _filter_already_selected(self, modules, pkgdict):
     def filter_already_selected(self, modules):
         self._filter_already_selected(modules, self.recommends)
 
+    def tocompose(self, prefix, arch, ignore_broken=False, comment=None):
+        packages = self.solved_packages.get(arch, dict())
+
+        name = self.name
+
+        missing = dict()
+        if arch == '*':
+            missing = self.not_found
+
+        content = ''
+
+        unresolvable = self.unresolvable.get(arch, dict())
+        for name in sorted(list(packages) + list(missing) + list(unresolvable)):
+            if name in self.silents:
+                continue
+            if name in missing:
+                if ignore_broken and name not in self.required:
```

Comment:
Probably needs to log the error if `ignore_broken` is false, like `toxml`?

Reply:
it would be a second time for same problem atm, but yes, we would need to move the error from the group generator when we drop that code. 

I could add it there, but comment it out for now...

Reply:
IMO both output formats should be independent anyway, as we'll have to support both for the time being.

Reply:
it is logging twice atm